NitroxServer/Serialization/SaveDataUpgrades/Upgrade_V1700.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Newtonsoft.Json.Linq;
+using NitroxServer.Serialization.Upgrade;
+
+namespace NitroxServer.Serialization.SaveDataUpgrades
+{
+    public class Upgrade_V1700 : SaveDataUpgrade
+    {
+        public override Version TargetVersion { get; } = new Version(1, 7, 0, 0);
+
+        protected override void UpgradePlayerData(JObject data)
+        {
+            foreach (JToken playerEntry in data["Players"])
+            {
+                playerEntry["HiddenSignalPings"] = new JArray();
+            }
+        }
```

Comment:
This should not work as the new field in [PersistedPlayerData.cs](https://github.com/SubnauticaNitrox/Nitrox/pull/1810/files#diff-6fd93b15a768ba981f6f0524d52a22d69d3ea35dac8994a320f1689bad2a4028) is called `PingInstancePreferences`, or am I wrong?

Reply:
yeah I forgot to delete that file